### PR TITLE
[coreimage] Add new API from iOS 10 beta 1

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by spouliot** [comment]
@dalexsoto would you review please ? thanks :)

File: src/CoreImage/CIContext.cs [hunk 1]
```diff
@@ -104,9 +99,35 @@ public CIContextOptions (NSDictionary dictionary)
...
+			set {
+				SetBooleanValue (CIContext.CacheIntermediates, value);
+			}
+		}
 	}
 	
 	public partial class CIContext {
+
+		public CIContext (CIContextOptions options) :
+			this (options == null ? null : options.Dictionary)
```
Comment (dalexsoto) [inline src/CoreImage/CIContext.cs]: Super minor, Syntax sugar, I think you can just do `this (options?.Dictionary)`

**Comment by dalexsoto** [comment]
LGTM 👍

File: src/CoreImage/CIContext.cs (hunk 1 above)
Comment (spouliot) [inline src/CoreImage/CIContext.cs]: indeed, I'm wasting bits :-)

**Comment by xamarin-release-manager** [comment]
Build success

**Pull request merged**